client/src/pages/Login/LoginForm/useStyles.ts

```diff
@@ -27,7 +29,7 @@ const useStyles = makeStyles((theme) => ({
     borderRadius: theme.shape.borderRadius,
     marginTop: 49,
     fontSize: 16,
-    backgroundColor: '#3a8dff',
+    backgroundColor: '#f14140',
```

Comment:
Please use the MUI theme to set and apply colours to various components. By using a theme we can ensure that consistent styles are applied across the app :)